py3status/modules/weather_owm.py

```diff
@@ -0,0 +1,587 @@
+(and free!) Weather Icons font (https://erikflowers.github.io/weather-icons),
+which has icons for most weather scenarios.
+
+For more information, see the documentation
+(ttps://openweathermap.org/weather-conditions) on what weather conditions are
+supported. See the configuration options for how to specify each weather icon.
+
+Configuration parameters:
+    api_key: Your OpenWeatherMap API key.
+        See https://openweathermap.org/appid
+        (default None)
+    cache_timeout: The time between weather polling in seconds
+        It is recommended to keep this at a higher value to avoid rate
+        limiting with the API's.
+        (default 600)
+    forecast_format: Formatting for future forecasts
+        Available placeholders:
+            icon, clouds, snow, wind, humidity, pressure, temp,
+            desc, desc_long
+        This is similar to the 'format' field, but contains information
+        for future weather
+        (default '{icon}')
+    forecast_num: Number of days to include in the forecast, including today
+        (regardless of the 'forecast_today' flag)
+        (default 0)
+    forecast_separator: Separator between entries in the forecast
+        (default ' ')
+    forecast_today: Include today in the forecast? (Boolean)
+        (default False)
+    format: How to display the weather. This also dictates the type of
+        forecast. The placeholders here refer to the format_[...] variables
+        found below.
+        Available placeholders:
+            icon, clouds, rain, snow, wind, humidity, pressure, temp, sunrise,
+            sunset desc, desc_long, forecast
+        You may also use the icons in the icon dictionary with their identifiers
+        (default '{icon}: {temp}')
+    format_clouds: Formatting for cloud coverage (percentage).
+        Available placeholders:
+            icon, coverage
+        (default '{icon}: {coverage}%')
+    format_humidity: Formatting for humidity (percentage)
+        Available placeholders:
+            icon, humid
+        (default '{icon}: {humid}%')
+    format_pressure: Formatting for atmospheric pressure
+        Available placeholders:
+            icon, press, sea_level
+        (default '{icon}: {press} hPa')
+    format_rain: Formatting for rain volume over the past 3 hours
+        Available placeholders:
+            icon, mm, cm, m, in, ft, yrd
+        (default '{icon}: {in} inches')
+    format_snow: Formatting for snow volume over the past 3 hours
+        Available placeholders:
+            icon, mm, cm, m, in, ft, yrd
+        (default '{icon}: {in} inches')
+    format_sunrise: Formatting for sunrise time.
+        Note that this format accepts strftime/strptime placeholders to populate
+        the output with the time information.
+        Available placeholders:
+            icon
+        (default '{icon}: %X')
+    format_sunset: Formatting for sunset time.
+        Note that this format accepts strftime/strptime placeholders to populate
```

Comment:
Opinion: I think we start off with "This" instead of "Note that this" in sunrise/sunset will be slightly nicer to parse. Very nice that we don't have `format_sun*_time` here anymore. :-)